django_extensions/db/fields/encrypted.py

```diff
@@ -24,30 +28,19 @@ class BaseEncryptedField(models.Field):
     prefix = 'enc_str:::'
 
     def __init__(self, *args, **kwargs):
-        warnings.warn(
-            "Please do not use these fields. Since Keyczar is deprecated and authors deleted "
-            "all code we highly recommend to stop using potentially unsafe abandonware directly "
-            "these versions of encrypted fields will be removed soon but it's highly likely their "
-            "names will be reused. Please see github issues/1359 for discussion on possible "
-            "replacement fields",
-            RemovedInNextVersionWarning,
-            stacklevel=2,
-        )
+        if not getattr(settings, 'CRYPTOGRAPHY_ENCRYPT_ALGORITHM', None):
+            raise ImproperlyConfigured('You must set the settings.CRYPTOGRAPHY_ENCRYPT_ALGORITHM '
+                                       'setting to your cryptography keys directory.')
```

Comment:
This message doesn't seem to match the setting

Reply:
We should also name this a bit different / less generic.

I would also don't mind using a dict in settings.py for configuration, specially if there is more things to parameterize.

For example (but by no means an official recommendation :-) : 
```
DJANGO_EXTENSIONS_ENCRYPTION_FIELDS = {
  "CRYPTOGRAPHY_ENCRYPT_ALGORITHM": "AES",
  "CRYPTOGRAPHY_ENCRYPT_KEY": os.urandom(16),
  "PBKDF2HMAC_ALGORITHM": "SHA256",
  "PBKDF2HMAC_LENGTH": 32,
  "SALT": os.urandom(16),
}

# or 

DJANGO_EXTENSIONS_ENCRYPTION_FIELDS_ALGORITHM = "AES"
```